build/package/automation_agent.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -ex
+
+PATH="$PATH":/bin:/usr/bin:/usr/local/bin
+
```

Comment:
I don't know why I don't see the spawn host in the cloud-manager UI. Could you have a look at this script and tell me if you find something wrong? I tested in my local machine the function `replace_property_in_file` and it works. @gssbzn